protocols/gossipsub/src/behaviour.rs

```diff
@@ -526,14 +526,14 @@ where
     /// Returns [`Ok(true)`] if the subscription worked. Returns [`Ok(false)`] if we were already
     /// subscribed.
     pub fn subscribe<H: Hasher>(&mut self, topic: &Topic<H>) -> Result<bool, SubscriptionError> {
-        tracing::debug!("Subscribing to topic: {}", topic);
+        tracing::debug!(%topic, "Subscribing to topic:");
         let topic_hash = topic.hash();
         if !self.subscription_filter.can_subscribe(&topic_hash) {
             return Err(SubscriptionError::NotAllowed);
         }
 
         if self.mesh.get(&topic_hash).is_some() {
-            tracing::debug!("Topic: {} is already in the mesh.", topic);
+            tracing::debug!(%topic, "Topic is already in the mesh.");
```

Comment:
Let's remove the `.` here.

```suggestion
            tracing::debug!(%topic, "Topic is already in the mesh");
```

Reply:
should i remove `.` at the end of all trace messages?